packages/react-searchbox/src/components/SearchComponent.js

```diff
@@ -199,16 +230,21 @@ class SearchComponent extends React.Component {
         this.setState(state);
       }, subscribeTo);
     }
-    if (value || customQuery) {
-      this.componentInstance.triggerCustomQuery();
-    }
   }
 
   componentDidMount() {
-    const { triggerQueryOnInit } = this.props;
-
-    if (triggerQueryOnInit) {
-      this.componentInstance.triggerDefaultQuery();
+    const { triggerQueryOnInit, customQuery } = this.props;
+    if (this.componentInstance) {
+      if (triggerQueryOnInit) {
+        this.componentInstance.triggerDefaultQuery();
+      }
+      if (
```

Comment:
@mohdashraf010897 This logic is new. We were not fetching custom query.

Reply:
No, this logic isn't new. the logic at [line 202](https://github.com/appbaseio/searchbox/pull/165/files#diff-10723a7edcee81321f7b746503aa676a0b7db2ec10743bb7d2982d79ff7bbfdeL202) is moved inside `componentDidMount` lifecycle method.

<img width="757" alt="image" src="https://user-images.githubusercontent.com/57627350/165623169-d6a5caf8-dac5-4dbc-8f62-03ebaebbc326.png">